Source/Serilog.Enrichers.Span/Serilog.Enrichers.Span.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup Label="Build">
-    <TargetFrameworks>net5.0;netcoreapp3.1;netcoreapp3.0</TargetFrameworks>
+    <TargetFrameworks>net472;net5.0;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
```

Comment:
Can we move net472 to the end, since we still want the default to be the latest `net5.0`.

Reply:
Sure, that is where Rider put it.

Reply:
Looks like some other files have crept in.

Reply:
Oh shoot, sorry I was rushing and used the command line. Normally the jetbrains files are ignored in my repos :)

I also squashed the commit to keep the history clean.